apps/chat/src/components/Chat/Migration/MigrationFailedModal.tsx

```diff
@@ -203,14 +225,15 @@ export const MigrationFailedWindow = ({
     promptsToRetryIds,
   ]);
 
-  const onRetryWithoutBackup = useCallback(() => {
-    retryMigration();
-  }, [retryMigration]);
+  const onBackupPrompts = useCallback(() => {
+    dispatch(ImportExportActions.exportLocalStoragePrompts());
+    BrowserStorage.setEntityBackedUp(MigrationStorageKeys.PromptsBackedUp);
+  }, [dispatch]);
 
-  const onRetryWithBackup = useCallback(() => {
-    dispatch(ImportExportActions.exportLocalStorageEntities());
-    retryMigration();
-  }, [dispatch, retryMigration]);
+  const onBackupChats = useCallback(() => {
```

Comment:
nit: as I know in react it's usually handleSomething is method inside same component and onSomething is method which passed to current component
What do you think about rename them?